src/js/components/FormField/FormField.js

```diff
@@ -32,12 +32,7 @@ const grommetInputNames = [
   'FileInput',
   'RadioButtonGroup',
 ];
-const grommetInputPadNames = [
-  'CheckBox',
-  'CheckBoxGroup',
-  'RadioButtonGroup',
-  'RangeInput',
-];
+const grommetInputPadNames = ['CheckBox', 'CheckBoxGroup', 'RadioButtonGroup'];
```

Comment:
It seems like removing this here is causing snapshot changes for existing storybooks since base.js cause `formField.content.pad = 'small'` by default. Can we leave RangeInput in this array but just add the pad option like you've done below to enable themes like grommet-theme-hpe to specify pad directly on the RangeInput since we override the formfield.content.pad to undefined?

Reply:
opps this was not meant to be changed 
part of me playing around with things